man/rustc.1

```diff
@@ -242,6 +242,52 @@ full debug info with variable and type information.
 \fBopt\-level\fR=\fIVAL\fR
 Optimize with possible levels 0\[en]3
 
+.SH ENVIRONMENT VARIABLES
+
+.TP
+\fBRUST_TEST_TASKS\fR
+The test framework Rust provides executes tests in parallel. This variable sets
+the maximum number of threads used for this purpose.
+
+.TP
+\fBRUST_TEST_NOCAPTURE\fR
+A synonym for the --nocapture flag.
+
+.TP
+\fBRUST_REGION_GRAPH_NODE\fR
+Works with `-Z print-region-graph`. Only produces output for the node ID provided
+via this variable.
```

Comment:
This seems like a somewhat unstable option, so perhaps it (and `RUST_REGION_GRAPH` below) could be left out?